### ISSUE TITLE
Iconfont Configuration Doesn't Reflect the Icon on Nuxt

### ISSUE THREAD

**Issue by jefrydco** [vuetifyjs/nuxt#68]
### Versions and Environment
Vuetify: 1.2.6
Vue: 2.5.17
Browsers: 70.0.3538.67 (Official Build) (64-bit)
OS: Fedora 28

### Previously worked in:
Vuetify:
Vue:

### Steps to reproduce

Add iconfont configuration as nuxt plugin.

### Expected Behavior

Iconfont configuration reflects the icon.

### Actual Behavior

Iconfont configuration doesn't reflect the icon.

### Reproduction Link

https://codesandbox.io/s/github/jefrydco/vuetify-mdi-icon

---

**Comment by Codefa** [comment]
@jefrydco its easy to fix just put your mdi-icon-name inside v-icon tag like this 

### for menu
`<v-icon>mdi-view-headline</v-icon>`

**Comment by Codefa** [comment]
@jefrydco to fix thi atleast for now you need to install a dependency " material-design-icons-iconfont "
and import it to the plugins it will works

**Comment by jefrydco** [comment]
@Codefa Take a look this https://github.com/vuetifyjs/vuetify/blob/dev/src/components/Vuetify/mixins/icons.js, I thought the `mdi` value for iconfont should be fine if I just add the `@mdi/font` packages.

And the `material-design-icons-iconfont` dependency actually the default icon used in vuetify. On this repo that package is loaded manually through the head section.

**Comment by Codefa** [comment]
@jefrydco another way is use the mdi-iconName inside the v-icon tag without adding material-design-icons-iconfont don't know why this works as per the docs in the official site
`<v-icon>mdi-menu</v-icon>`

**Comment by jefrydco** [comment]
@Codefa Yup, actually including `@mdi/font` package will be worked by adding prefix to icon name, but one thing I consider is, the default icon used by vuetify like hamburger menu doesnt change into mdi-prefixed after I configure iconfont to mdi.
